AnnotatoBackend/Sources/App/WebSocketControllers/WebSocketController.swift

```diff
@@ -0,0 +1,108 @@
+import Foundation
+import Vapor
+import AnnotatoSharedLibrary
+import FluentKit
+
+class WebSocketController {
+    /// Each user has one websocket connection.
+    private static var openConnections: [String: WebSocket] = [:]
+    private static var logger = Logger(label: "WebSocketController")
+
+    static func handleIncomingConnection(req: Request, webSocket: WebSocket) {
+        guard let userId = try? ControllersUtil.getIdFromParamsAsString(request: req) else {
+            req.logger.info("Could not get user ID for request. Closing WebSocket connection...")
+            _ = webSocket.close()
+            return
+        }
+
+        req.logger.info("Client with ID: \(userId) connected!")
+        addToOpenConnections(userId: userId, webSocket: webSocket)
```

Comment:
Should we handle the case where a user already has an open connection, in which case we should perhaps immediately close the new connection instead of letting it replace the old connection?